packages/admin/dashboard/src/i18n/translations/$schema.json

```diff
@@ -258,7 +258,11 @@
               "type": "string"
             }
           },
-          "required": ["header_one", "header_other", "description"],
+          "required": [
```

Comment:
@kasperkristensen, this file keeps getting updated because of linting 😄 

Not a big deal, but anything we can do here? It clutters the PRs a bit